### PR TITLE
added Input toggle functions

### PR DESCRIPTION
**Pull request by HiDefDan**
Basic input toggle function added using the inputs available on RZ21K / RZ31K projectors.
Tested on projectors with password cleared
#6 @JeffreyDavidsz

### CODE REVIEW

File: pjlink.js [hunk 1]
```diff
@@ -225,14 +225,34 @@ instance.prototype.actions = function(system) {
 		'shutterOpen':    { label: 'Open Shutter' },
 		'shutterClose':   { label: 'Close Shutter' },
 		'freeze':         { label: 'Freeze Input' },
-		'unfreeze':       { label: 'Unfreeze Input' }
-
+		'unfreeze':       { label: 'Unfreeze Input' },
+		'inputToggle': {
+			label: 'Toggle Input',
```
Comment (haakonnessjoen) [inline pjlink.js:230]: Isn't "change input", "set input" or "select input" better name? Toggling usually infers that it is a boolean choice.

Comment (oschuetze) [reply pjlink.js:230]: The [specification](https://pjlink.jbmia.or.jp/english/data_cl2/PJLink_5-1.pdf) calls the command "Input switch" - this should be used in my opinion. I also would use "toggle" for a boolean choice only.

**Comment by haakonnessjoen** [comment]
@HiDefDan could you please change the naming, and I will merge it

**Comment by HiDefDan** [comment]
All done hopefully - my git fu is a bit rusty after 3 months furloughed / locked down being a stay home dad ;)

**Comment by HiDefDan** [comment]
@haakonnessjoen please look at the other open pull request  - my git fu is not what it could be but I think that the changes requested are done there

**Pull request closed without merging**